src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs

```diff
@@ -6,38 +6,34 @@
 namespace Microsoft.Extensions.Logging.EventSource
 {
     /// <summary>
-    /// The LoggingEventSource is the bridge form all ILogger based logging to EventSource/EventListener logging.
+    /// The LoggingEventSource is the bridge form all `ILogger`-based logging to EventSource/EventListener logging.
     ///
-    /// You turn this logging on by enabling the EvenSource called
-    ///
-    ///      Microsoft-Extensions-Logging
+    /// You turn this logging on by enabling the EventSource called `Microsoft-Extensions-Logging`
     ///
     /// When you enabled the EventSource, the EventLevel you set is translated in the obvious way to the level
-    /// associated with the ILogger (thus Debug = verbose, Informational = Informational ... Critical == Critical)
+    /// associated with the ILogger (thus Debug = verbose, Informational = Informational ... Critical = Critical)
     ///
     /// This allows you to filter by event level in a straighforward way.
     ///
-    /// For finer control you can specify a EventSource Argument called
-    ///
-    /// FilterSpecs
+    /// For finer control you can specify an EventSource argument called `FilterSpecs`
     ///
-    /// The FilterSpecs argument is a semicolon separated list of specifications.   Where each specification is
+    /// The FilterSpecs argument is a semicolon-separated list of specifications, where each specification is
     ///
-    /// SPEC =                          // empty spec, same as *
-    ///      | NAME                     // Just a name the level is the default level
-    ///      | NAME : LEVEL            // specifies level for a particular logger (can have a * suffix).
+    ///     SPEC =                          // empty spec, same as *
+    ///          | NAME                     // Just a name the level is the default level
+    ///          | NAME : LEVEL             // specifies level for a particular logger (can have a * suffix).
     ///
-    /// Where Name is the name of a ILoggger (case matters), Name can have a * which acts as a wildcard
-    /// AS A SUFFIX.   Thus Net* will match any loggers that start with the 'Net'.
+    /// **NAME** is the name of a ILogger (case matters), Name can have a * which acts as a wildcard
+    /// *as a suffix*. Thus, `Net*` will match any loggers that start with the string 'Net'.
     ///
```

Comment:
These empty lines don't really do anything - they are ignored, just like in HTML.

To have paragraph breaks, each paragraph has to be wrapped in `<para>...</para>` tags, like this:

```xml
        /// <para>
        /// Para1
        /// </para>
        /// <para>
        /// Para2
        /// </para>
```

Reply:
I've made the changes. b9094e57db52cb1bfc512b66246c4befcc467d09 has the final version.

Let me know if I still didn't understand what you meant, or if you think I'm still not understanding how the docs system pulls out the xmldoc and generates the reference page (which I believe shows up at https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.extensions.logging.eventsourcelogger)